<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="vofking-Aspire-R7-372T" ignored="0" name="Default test" tests="10" failures="10" timestamp="2020-05-18T01:38:21 PDT" time="0.525" errors="0">
  <testcase name="test" time="0.094" classname="core.PrimeNumberCheckerTestDR">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;lt;false&amp;gt;
     but: was &amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: <false>
     but: was <true>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTestDR.test(PrimeNumberCheckerTestDR.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.01" classname="core.PrimeNumberCheckerTestDR">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;lt;false&amp;gt;
     but: was &amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: <false>
     but: was <true>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTestDR.test(PrimeNumberCheckerTestDR.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.013" classname="core.PrimeNumberCheckerTestDR">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;lt;true&amp;gt;
     but: was &amp;lt;false&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: <true>
     but: was <false>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTestDR.test(PrimeNumberCheckerTestDR.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.004" classname="core.PrimeNumberCheckerTestDR">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;lt;false&amp;gt;
     but: was &amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: <false>
     but: was <true>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTestDR.test(PrimeNumberCheckerTestDR.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.02" classname="core.PrimeNumberCheckerTestDR">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;lt;true&amp;gt;
     but: was &amp;lt;false&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: <true>
     but: was <false>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTestDR.test(PrimeNumberCheckerTestDR.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.005" classname="core.PrimeNumberCheckerTestDR">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;lt;false&amp;gt;
     but: was &amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: <false>
     but: was <true>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTestDR.test(PrimeNumberCheckerTestDR.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.016" classname="core.PrimeNumberCheckerTestDR">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;lt;true&amp;gt;
     but: was &amp;lt;false&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: <true>
     but: was <false>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTestDR.test(PrimeNumberCheckerTestDR.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.005" classname="core.PrimeNumberCheckerTestDR">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;lt;true&amp;gt;
     but: was &amp;lt;false&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: <true>
     but: was <false>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTestDR.test(PrimeNumberCheckerTestDR.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.007" classname="core.PrimeNumberCheckerTestDR">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;lt;true&amp;gt;
     but: was &amp;lt;false&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: <true>
     but: was <false>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTestDR.test(PrimeNumberCheckerTestDR.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.012" classname="core.PrimeNumberCheckerTestDR">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;lt;false&amp;gt;
     but: was &amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: <false>
     but: was <true>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTestDR.test(PrimeNumberCheckerTestDR.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- test -->
</testsuite> <!-- Default test -->
